extensions/binary_glTF/README.md

```diff
@@ -1,10 +1,12 @@
-# CESIUM_binary_glTF
+# binary_glTF
```

Comment:
Instead of changing it back, I think we should just leave the `CESIUM_binary_glTF` extension as is, and then copy and paste it to create the `binary_glTF` extension in a new directory.

I plan to move Cesium to `binary_glTF` as fast as possible - end of September or end of October.  Leaving the Cesium-specific extension around for now should have no impact on our official extension.